packages/utilities/gel-foundations/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 3.3.2 | [PR#1758](https://github.com/bbc/psammead/pull/1758) Update Thai script type sizes for Minion and Brevier. |
```

Comment:
the package.json says 3.2.2 ... looks like a previous PR bumped the CHANGELOG higher than the package.json

Reply:
Said that, is this a breaking change? Should we be updating the minor version instead?

Reply:
Good spot! It was a typo. Fixed it now. https://www.npmjs.com/package/@bbc/gel-foundations

Reply:
@DenisHdz I'd say it'd count as a bug fix - since the previous values were incorrect. So that's why I classed it as a patch version bump.

Reply:
In that case, LGTM :)